os_builders/playbooks/run_vm_baseline.yml

```diff
@@ -0,0 +1,5 @@
+---
+- name: run VM_baseline
+  hosts: localhost
+  roles:
+    - role: vm_baseline
```

Comment:
Could we use the existing playbook? Since that has some safety guard but should work the same as this one

Reply:
done